Compiler panics with property assignments of globals from external file in init callback

## Steps to reproduce

Put the following files in the slint-cpp-template:

`appwindow.slint`:
```
import { Theme, SubTheme } from "./theme.slint";

// Interestingly, if I move the globals Theme and SubTheme here, it won't crash

export component AppWindow inherits Window {
    init => {
        Theme.font-family = SubTheme.font-family;
    }
}

```

`theme.slint`:
```
export global Theme {
    in-out property <string> font-family: "Arial";
}

export global SubTheme {
    in-out property <string> font-family: "Consolas" ;
}
```

(You can clone it from https://github.com/slint-ui/slint-cpp-template/compare/main...MiKom:slint-cpp-template:init-global-panic)
Tested on 0.3.5

## Expected result

Project compiles and works

## Actual result

When I build, Slint compiler crashes with the following output:
```
PS C:\Users\mikom\Documents\projects\slint\slint-cpp-template> cmake --build .\build
[0/1] Re-running CMake...-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/mikom/Documents/projects/slint/slint-cpp-template/build

[1/3] Generating appwindow.h
FAILED: appwindow.h C:/Users/mikom/Documents/projects/slint/slint-cpp-template/build/appwindow.h
cmd.exe /C "cd /D C:\Users\mikom\Documents\projects\slint\slint-cpp-template\build && "C:\Program Files\Slint-cpp 0.3.5\bin\slint-compiler.exe" C:/Users/mikom/Documents/projects/slint/slint-cpp-template/ui/appwindow.slint -o C:/Users/mikom/Documents/projects/slint/slint-cpp-template/build/appwindow.h --style native"
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', internal\compiler\llr\lower_to_item_tree.rs:94:15
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
ninja: build stopped: subcommand failed.
```

--- a/internal/compiler/object_tree.rs
+++ b/internal/compiler/object_tree.rs
@@ -2056,6 +2056,11 @@ pub fn visit_all_named_references(
             compo
         },
     );
+    component
+        .init_code
+        .borrow_mut()
+        .iter_mut()
+        .for_each(|expr| visit_named_references_in_expression(expr, vis));
 }
 
 /// Visit all expression in this component and sub components

--- a/tests/cases/callbacks/init.slint
+++ b/tests/cases/callbacks/init.slint
@@ -3,6 +3,8 @@
 
 // Verify that the init callback is invoked in the correct order
 
+import { ExportedGlobal } from "../../helper_components/export_globals.slint";
+
 export global InitOrder := {
     property <string> observed-order: "start";
 }
@@ -45,6 +47,7 @@ TestCase := Rectangle {
     height: 300phx;
     init => {
         InitOrder.observed-order += "|root";
+        ExportedGlobal.foo += 1;
     }
     Sub1 {
         init => {

